apps/generic_terminal/generic_terminal.talon

```diff
@@ -8,7 +8,7 @@ lisa all:
     user.terminal_list_all_directories()
 katie [<user.text>]: user.terminal_change_directory(text or "")
 katie root: user.terminal_change_directory_root()
-go <user.system_path>: insert("cd \"{system_path}\"\n")
+go <user.system_path>: insert("cd {system_path} \n")
```

Comment:
This is a bad idea, as it means any path that contains spaces (or other characters with special meaning to the shell) will not work without escaping. Instead, I suggest making a capture `<user.system_path>` which calls `os.expanduser` on the path.